src/ServiceControl.Config/UI/InstanceAdd/ServiceControlAddAttachment.cs

```diff
@@ -85,7 +85,6 @@ async Task Add(object arg)
                 HostName = viewModel.ServiceControlAudit.HostName,
                 Port = Convert.ToInt32(viewModel.ServiceControlAudit.PortNumber),
                 DatabaseMaintenancePort = Convert.ToInt32(viewModel.ServiceControlAudit.DatabaseMaintenancePortNumber),
-                VirtualDirectory = null, // TODO
```

Comment:
So the reason to remove this is

> This setting is provided for backward compatibility and should be considered obsolete.

https://docs.particular.net/servicecontrol/creating-config-file#host-settings-servicecontrolvirtualdirectory

so should it be removed completely then (happy for a different PR)

Reply:
I'm happy to remove that (and some other code around the virtual directory thingy) altogether in a separate PR.